src/Bicep.LangServer/Extensions/DiagnosticExtensions.cs

```diff
@@ -30,6 +31,14 @@ private static DiagnosticSeverity ToDiagnosticSeverity(DiagnosticLevel level)
                 DiagnosticLevel.Error => DiagnosticSeverity.Error,
                 _ => throw new ArgumentException($"Unrecognized level {level}"),
             };
+
+        private static Container<DiagnosticTag>? ToDiagnosticTags(DiagnosticLabel? label) => label switch
+        {
+            null => null,
+            DiagnosticLabel.Unnecessary => new Container<DiagnosticTag>(DiagnosticTag.Unnecessary),
+            DiagnosticLabel.Deprecated => new Container<DiagnosticTag>(DiagnosticTag.Deprecated),
```

Comment:
This is how `DiagnosticTag.Deprecated` is rendered in VSCode:
![image](https://user-images.githubusercontent.com/16367959/109738570-e69ad500-7b7c-11eb-9653-64252efaeb34.png)